### DISCUSSION TITLE
Ark Plus: License page is 404-ing

### DISCUSSION THREAD

**Discussion by Naoto-Ida** [chakra-ui/ark#2897] [Q&A] [answered]
Hey y'all, we're considering [Ark UI Plus](https://ark-ui.com/react/plus), but the [licensing page](https://ark-ui.com/license) is 404-ing. Is there some alternative page that I could read? Thanks.

**Comment by segunadebayo** [comment, marked as answer]
Hey @Naoto-Ida,

I've updated the docs to include the license agreement.
